lib/metasploit/framework/login_scanner/http.rb

```diff
@@ -348,6 +324,38 @@ def attempt_login(credential)
           Result.new(result_opts)
         end
 
+
+        # This method returns a proc to log HTTP requests and responses
+        # if datastore['HttpTrace'] is set.
```

Comment:
Just something I noticed but we are missing RDoc information for some of these functions and I think we should be adding these in considering some of the methods here appear to have RDoc documentation and other's don't. Particularly for these new methods of your own that you are adding in, you should be looking to add some RDoc headers in given you have the info to do this :)